GPyOpt/util/arguments_manager.py

```diff
@@ -51,6 +51,7 @@ def acquisition_creator(self, acquisition_type, model, space, acquisition_optimi
         cost_withGradients = cost_withGradients
         acquisition_jitter = self.kwargs.get('acquisition_jitter',0.01)
         acquisition_weight = self.kwargs.get('acquisition_weight',2)
+        QMC_iterations = self.kwargs.get('N_QMC',16)
```

Comment:
This needs a docstring I guess

Reply:
Is it enough to have it in the NEI.py file or does it need to be somewhere else?

Reply:
Unfortunately it is a bit tangled up in GPyOpt, so you want to have it in this file as well.